views/pages/search.html

```diff
@@ -32,7 +32,7 @@
           <img
             src="{{baseUrl}}/public/images/system-number-hint.png"
             alt="Where to find the system number on a birth certificate"
-            title="Where to find the system number on a birth certificate"
+            title="The system number is a nine digit number located in the bottom left corner of the birth certificate"
```

Comment:
One of these accessibility guys tested this in a screen reader and it only read out the alt text. So perhaps we should swap the two around?